src/common/universalRouterDeployments.ts

```diff
@@ -17,4 +17,5 @@ export const universalRouterDeploymentByChainId: Record<ChainId, LiquiditySource
     [ChainId.BaseMainnet]: revision1,
     [ChainId.BlastMainnet]: revision1,
     [ChainId.SuiMainnet]: notDeployed,
+    [ChainId.UnichainMainnet]: revision1,
```

Comment:
Should be `notDeployed` as universal router contract was not deployed to unichain yet